triton-profiler/src/triton_profiler.rs

```diff
@@ -75,9 +77,8 @@ impl TritonProfiler {
             self.stack.is_empty(),
             "Cannot generate report before stack is empty."
         );
-        assert_ne!(
-            Duration::ZERO,
-            self.total_time,
+        assert!(
+            self.total_time != Duration::ZERO,
```

Comment:
```suggestion
        assert_ne!(
            self.total_time,
            Duration::ZERO,
```

Reply:
We usually do `assert_eq!(desired, actual)`. I'm not quite sure how that translates to `assert_ne!(…, …)` but usually go with `assert_ne!(undesired, actual)`.